Diagnostics/HealthChecker/DataCollection/ServerInformation/Get-OperatingSystemInformation.ps1

```diff
@@ -90,6 +115,6 @@ Function Get-OperatingSystemInformation {
     $osInformation.Smb1ServerSettings.WindowsFeature = $getSmb1ServerSettings.WindowsFeature
     $osInformation.Smb1ServerSettings.Smb1Status = $getSmb1ServerSettings.Smb1Status
```

Comment:
This is no longer available because of the modified logic in `Get-Smb1ServerSettings`

Reply:
`int Smb1Status` is also still present in `public class Smb1ServerSettings`